profiles/functions

```diff
@@ -474,13 +474,13 @@ restore_logs_syncing() {
 }
 
 irqbalance_banned_cpus_clear() {
-    sed -i '/^IRQBALANCE_BANNED_CPUS=/d' /etc/sysconfig/irqbalance
+    sed -i '/^IRQBALANCE_BANNED_CPUS=/d' /etc/sysconfig/irqbalance && systemctl try-restart irqbalance
```

Comment:
Notice that due to this change, `irqbalance` will be restarted twice in a row when `irqbalance_banned_cpus_setup` is called. This seems like unnecessary overhead and it might occasionally fail - for some reason, systemd doesn't like restarting the same service multiple times in quick succession.

Reply:
Right, in that case I guess a proper "irqbalance" plugin (as requested in BZ1784645) might be a better idea that does the restart only once.  Thoughts?

Reply:
@olysonek , would something like this work for you until we get a proper `irqbalance` plugin?
```
irqbalance_banned_cpus_clear() {
    sed -i '/^IRQBALANCE_BANNED_CPUS=/d' /etc/sysconfig/irqbalance && \
        test ${1:-restart} = restart && systemctl try-restart irqbalance
}

irqbalance_banned_cpus_setup() {
    irqbalance_banned_cpus_clear norestart
    if [ -n "$1" ]; then
        echo "IRQBALANCE_BANNED_CPUS=$1" >> /etc/sysconfig/irqbalance
    fi
    systemctl try-restart irqbalance
}
```

Reply:
> @olysonek , would something like this work for you until we get a proper `irqbalance` plugin?

Yes, that's what I had in mind.

Reply:
Thanks!  Pushed an update.